bitnami/etcd/values.yaml

```diff
@@ -129,10 +129,13 @@ auth:
 ## Kubernetes Security Context
 ## https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
 ##
+securityContextEnabled: true
 securityContext:
-  enabled: true
```

Comment:
Please revert this change. It includes `backwards` incompatibilities.

Reply:
Sorry could you elaborate? How is it incompatible?

Reply:
It modifies the structure of the **values.yaml**. 

It's true it doesn't imply a major change but a user who's using a specific **values.yaml** needs to adapt to continue enabling the securityContext. Therefore, if we want to go ahead with the change (at least) we need to mention the change in the README.md (Upgrading section) to alert users about the change.